Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -324,69 +327,91 @@ export async function insertOutboundReferences(
 }
 
 /**
- * Inserts or updates a document in the database
- * @param param0 Document info for insert/update
- * @param isInsert is insert or update SQL re
- * @returns if the result returned rows
+ * Updates a document in the database
  */
-export async function insertOrUpdateDocument(
+export async function updateDocument(
   client: PoolClient,
-  { meadowlarkId, documentUuid, resourceInfo, documentInfo, edfiDoc, validateDocumentReferencesExist, security },
-  isInsert: boolean,
-): Promise<boolean> {
-  const documentValues = [
+  {
     meadowlarkId,
     documentUuid,
-    JSON.stringify(documentInfo.documentIdentity),
-    resourceInfo.projectName,
-    resourceInfo.resourceName,
-    resourceInfo.resourceVersion,
-    resourceInfo.isDescriptor,
-    validateDocumentReferencesExist,
-    security.clientId,
+    resourceInfo,
+    documentInfo,
     edfiDoc,
-  ];
+    validateDocumentReferencesExist,
+    security,
+  }: UpdateRequest,
+): Promise<boolean> {
+  const queryResult: QueryResult<any> = await client.query(
+    `
+      UPDATE meadowlark.documents
+        SET
+        meadowlark_id = $1,
+        document_identity = $2,
+        project_name = $3,
+        resource_name = $4,
+        resource_version = $5,
+        is_descriptor = $6,
+        validated = $7,
+        created_by = $8,
+        edfi_doc = $9,
+        last_modified_at = $10
+        WHERE meadowlark.documents.document_uuid = $11;`,
+    [
+      meadowlarkId,
+      JSON.stringify(documentInfo.documentIdentity),
+      resourceInfo.projectName,
+      resourceInfo.resourceName,
+      resourceInfo.resourceVersion,
+      resourceInfo.isDescriptor,
+      validateDocumentReferencesExist,
+      security.clientId,
+      edfiDoc,
+      documentInfo.requestTimestamp,
+      documentUuid,
+    ],
+  );
 
-  let documentSql: string;
+  return hasResults(queryResult);
+}
 
-  if (isInsert) {
-    documentSql = format(
-      `
+/**
+ * Inserts a document in the database
+ */
+export async function insertDocument(
+  client: PoolClient,
+  {
+    meadowlarkId,
+    documentUuid,
+    resourceInfo,
+    documentInfo,
+    edfiDoc,
+    validateDocumentReferencesExist,
+    security,
+  }: UpdateRequest,
+): Promise<boolean> {
+  const queryResult: QueryResult<any> = await client.query(
+    `
       INSERT INTO meadowlark.documents
         (meadowlark_id, document_uuid, document_identity, project_name, resource_name, resource_version, is_descriptor,
-        validated, created_by, edfi_doc)
-        VALUES (%L)
+        validated, created_by, edfi_doc, created_at, last_modified_at)
+        VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12)
         RETURNING document_uuid;`,
```

Comment:
pg-format treats numeric parameters like strings, so switched to node-postgres built-in.